roles/content_rhel/README.md

```diff
@@ -0,0 +1,123 @@
+theforeman.foreman.content_rhel
+===================================
+
+This role is an opinionated reuse of other roles in the collection, which creates a basic configuration for everything Katello needs to register and patch existing RHEL clients.
```

Comment:
```suggestion
This role is an opinionated reuse of other roles in the collection, which creates a basic configuration for everything needed to register and patch existing RHEL clients.
```

we try to avoid Foreman/Katello names to make branding easier :)